features/dashboard/lib/src/adding_report/ui/mobile/forest_adding_screen_mobile.dart

```diff
@@ -30,18 +30,30 @@ class ForestAddingScreenMobile extends StatefulWidget {
 
 class _ForestAddingScreenMobileState extends State<ForestAddingScreenMobile> {
   List<Uint8List> _selectedImages = [];
+  int imageSizes = 0;
 
   Future<void> getMultipleImageInfos() async {
     final images = await AppImagePicker().pickMultipleImages();
 
     setState(() {
+      imageSizes = 0;
       _selectedImages = (_selectedImages + images)
           .take(GlobalConstants.maxAllowedImageCount)
           .toList();
+      for (var element in _selectedImages) {
```

Comment:
Do not duplicate this code. Create separate function for that. Additionally, I would recommend to keep only `isImagesSizeValid`, because `imageSizes` is not used in class.